lib/runtime/wasmer/imports.go

```diff
@@ -2090,8 +2132,26 @@ func ext_storage_root_version_1(context unsafe.Pointer) C.int64_t {
 
 //export ext_storage_root_version_2
 func ext_storage_root_version_2(context unsafe.Pointer, version C.int32_t) C.int64_t {
-	// TODO: update to use state trie version 1 (#2418)
-	return ext_storage_root_version_1(context)
+	logger.Trace("executing...")
+
+	instanceContext := wasm.IntoInstanceContext(context)
```

Comment:
Why is this needed in this PR?

Reply:
Looks like things work even without changing this function, removed it.